5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/LearningJavaApplication.java

```diff
@@ -0,0 +1,47 @@
+package com.wizeline.maven.learningjava;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.InetSocketAddress;
+import java.net.URI;
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.function.Function;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+import org.json.JSONArray;
```

Comment:
Please remove all unused imports in this class

Reply:
ok